modules/core/src/main/java/org/apache/ignite/internal/client/GridClient.java

```diff
@@ -33,7 +33,6 @@
  * <p>
  * Use following methods to get access to remote cache functionality:
```

Comment:
For now, there is only one method which can be used to get access to cache functionality. Let's change the JavaDoc as follows: `Use the following method to get access to remote cache functionality`

Reply:
ok